db/create_analytics_views.sql

```diff
@@ -15,51 +15,192 @@
--- TODO: greeter_organization_join_records
--- TODO: greeter_roles
--- TODO: incoming_emails
--- TODO: incoming_text_messages
--- TODO: intakes
-CREATE OR REPLACE VIEW analytics.intakes as
-    SELECT id from intakes;
--- TODO: notes
--- TODO: organization_lead_roles
--- TODO: outbound_calls
--- TODO: outgoing_emails
--- TODO: outgoing_text_messages
--- TODO: provider_scrapes
--- TODO: signups
--- TODO: site_coordinator_roles
--- TODO: source_parameters
--- TODO: stimulus_triages
--- TODO: system_notes
--- TODO: tax_returns
--- TODO: team_member_roles
--- TODO: users
--- TODO: vita_partner_states
--- TODO: vita_partner_zip_codes
--- TODO: vita_partners
--- TODO: vita_providers
+    SELECT id, created_at, updated_at
+    FROM public.admin_roles;
+
+CREATE OR REPLACE VIEW analytics.anonymized_diy_intake_csv_extracts AS
+    SELECT id, created_at, record_count, run_at, updated_at
+    FROM public.anonymized_diy_intake_csv_extracts;
+
+CREATE OR REPLACE VIEW analytics.anonymized_intake_csv_extracts AS
+    SELECT id, created_at, record_count, run_at, updated_at
+    FROM public.anonymized_intake_csv_extracts;
+
+CREATE OR REPLACE VIEW analytics.client_success_roles AS
+    SELECT id, created_at, updated_at
+    FROM public.client_success_roles;
+
+CREATE OR REPLACE VIEW analytics.clients AS
+    SELECT id, attention_needed_since, created_at, current_sign_in_at, failed_attempts, first_unanswered_incoming_interaction_at, last_incoming_interaction_at, last_internal_or_outgoing_interaction_at, last_sign_in_at, locked_at, login_requested_at, routing_method, sign_in_count, updated_at, vita_partner_id
+    FROM public.clients;
+
+CREATE OR REPLACE VIEW analytics.coalition_lead_roles AS
+    SELECT id, coalition_id, created_at, updated_at
+    FROM public.coalition_lead_roles;
+
+CREATE OR REPLACE VIEW analytics.coalitions AS
+    SELECT id, created_at, name, updated_at
+    FROM public.coalitions;
+
+CREATE OR REPLACE VIEW analytics.delayed_jobs AS
+    SELECT id, attempts, created_at, failed_at, locked_at, locked_by, priority, queue, run_at, updated_at
+    FROM public.delayed_jobs;
+
+CREATE OR REPLACE VIEW analytics.dependents AS
+    SELECT id, created_at, disabled, intake_id, months_in_home, north_american_resident, on_visa, updated_at, was_married, was_student
+    FROM public.dependents;
+
+CREATE OR REPLACE VIEW analytics.diy_intakes AS
+    SELECT id, created_at, locale, referrer, source, updated_at, visitor_id, zip_code
+    FROM public.diy_intakes;
+
+CREATE OR REPLACE VIEW analytics.documents AS
+    SELECT id, client_id, contact_record_id, contact_record_type, created_at, document_type, documents_request_id, intake_id, tax_return_id, updated_at, uploaded_by_id, uploaded_by_type
+    FROM public.documents;
+
+CREATE OR REPLACE VIEW analytics.documents_requests AS
+    SELECT id, created_at, intake_id, updated_at
+    FROM public.documents_requests;
+
+CREATE OR REPLACE VIEW analytics.greeter_coalition_join_records AS
```

Comment:
oh wait... we actually have a greeter role? I didnt even know that. Rad.

Reply:
Yeah uh same